tests/index.test.ts

```diff
@@ -150,4 +151,48 @@ describe("Event handlers", () => {
         // should call addAnnotation on client
         expect(clientMock.addAnnotation).toHaveBeenCalledWith(annotation);
     });
+
+    it("should respond to emitted updateAnnotation event with handler", async () => {
+        const annotation = {
+            "@context": "fakeContext",
+            id: "newId",
+            body: {},
+            motivation: "commenting",
+            target: { source: "fakesource" },
+            type: "Annotation",
+        };
+        const previous = {
+            "@context": "oldfakeContext",
+            id: "oldId",
+            body: {},
+            motivation: "commenting",
+            target: { source: "oldfakesource" },
+            type: "Annotation",
+        };
+
+        clientMock.emit("updateAnnotation", annotation, previous);
+        // should call AdjustTargetSource
```

Comment:
@blms I can't figure out how to emit with two params. According to the tin-emitter docs I think this syntax is correct, but when the handler is called the second parameter is `undefined`. Help?

Reply:
Ah, I see the problem! It's an issue with my mock. I'll adjust the mock to work properly with multiple arguments. (I mocked it so I didn't have to rely on importing tiny-emitter itself and deal with any potential problems there)

Reply:
Should be fixed now

Reply:
I didn't even think to look in the local code! Thanks, would not have found that